src/Analyzers/CSharp/Tests/PopulateSwitch/PopulateSwitchExpressionTests.cs

```diff
@@ -962,7 +962,7 @@ void Method()
             (MyEnum)0 => 1,
             (MyEnum)1 => 2,
             ""Mismatching constant"" => 3,
-            _ => throw new System.NotImplementedException(),
+            MyEnum.FizzBuzz => throw new System.NotImplementedException(),
```

Comment:
When "Mismatching constant" was encountered, the analyzer considered the switch as "complete". Hence, it failed to 'add missing cases'.

The "mismatching constant" is a compile-error anyway. So it probably doesn't matter a lot.